conan/tools/cmake/cmake.py

```diff
@@ -44,9 +46,7 @@ class CMake(object):
     are passed to the command line, plus the ``--config Release`` for builds in multi-config
     """
 
-    def __init__(self, conanfile, generator=None, build_folder=None, parallel=True,
-                 msbuild_verbosity="minimal"):
-
+    def __init__(self, conanfile, generator=None, build_folder=None, parallel=True):
```

Comment:
So, in the "toolchain" build_helper version we drop that `msbuild_verbosity` parameter, right? (just double checking)

Reply:
Totally, both in the CMake and MSBuild helpers, that parameter is dropped. I don't see any value (actually a bit of noise) of specifying that in recipes

Reply:
Why dropping?
I would assume the configuration file provides a new default.
The search order of a value should be (from higher priority to lower priority):
1. explicitly set as argument
2. set in `[conf]` section of profile
3. set in `[conf]` section of global `conan_conf.txt`
4. default value, calculated by some implementation

Reply:
Isn't it very annoying, not to say impossible to edit the recipe (when building dependencies) to change something like the verbosity? So what is really wanted is the possibility to define in command line, for example, like ``-c tools.microsoft:msbuild_verbosity=Diagnostic``, but not editing the recipe.

Plus, defining it as an argument in the recipe, doesn't have that strong priority for all users. Many of them will still want to be able to change that from the downstream consumer, even if defined in code (so this would be in fact the lowest priority). So this is what we are priorizing now, the case of the real ``conf`` definition, and we plan to add it as argument when there is a stronger evidence of the use case and after discussing and agreeing on the priority.